code/game/objects/structures/shower.dm

```diff
@@ -227,23 +220,6 @@
 		L.adjustFireLoss(5)
 		to_chat(L, "<span class='danger'>[src] is searing!</span>")
 
-/obj/machinery/shower/proc/check_clothes(mob/living/carbon/human/H)
```

Comment:
Do we want to remove this clothing check too? It isn't *strictly* hygiene-related.

Reply:
Garen: im locked out of https://github.com/tgstation/tgstation/pull/44843 and i dont know space maniacs discaorc so i cant respond to him so can someone relay that the reason I removed check_clothes() was because it was only used with hygiene and was really hated as a part of it so i dont think it would be popular if it was reused

Reply:
Fine by me, in that case